django_project/lesson/templatetags/lesson_tags.py

```diff
@@ -14,3 +15,17 @@ def is_translation_up_to_date(value):
             '<span title="Translation is outdated"><sup>&#x2757;</sup></span>')
     else:
         return mark_safe('')
+
+
+
+@register.simple_tag(takes_context=True)
+def version_tag(context):
+    """Reads current project release from the .version file."""
+    version_file = absolute_path('.version')
+    try:
+        with open(version_file, 'r') as file:
+            version = file.read()
+            context['version'] = version
+    except:
```

Comment:
__[Flake8]__

[E722] do not use bare except'


<!-- Comment by [SideCI](https://sideci.com) -->